lab_1/main.py

```diff
@@ -4,56 +4,168 @@
 """
 
 
-def tokenize(text: str) -> list:
-    """
+#data = open('data.txt', encoding='utf-8').read()
+data = 'The weather is sunny, the man is happy. man man happy'
+def tokenize(text):
```

Comment:
please, return a function description and type annotation (everywhere)

def tokenize(text: str) -> list:
   """
    Splits sentences into tokens, converts the tokens into lowercase, removes punctuation
    :param text: the initial text
    :return: a list of lowercased tokens without punctuation
    e.g. text = 'The weather is sunny, the man is happy.'
    --> ['the', 'weather', 'is', 'sunny', 'the', 'man', 'is', 'happy']
   """
   YOUR CODE